libraries/core_libs/consensus/src/dag/dag.cpp

```diff
@@ -43,7 +43,6 @@ bool Dag::addVEEs(blk_hash_t const &new_vertex, blk_hash_t const &pivot, std::ve
   // add vertex
   vertex_t ret = add_vertex(new_vertex, graph_);
   boost::get(boost::vertex_index, graph_)[ret] = new_vertex;
-  edge_index_map_t weight_map = boost::get(boost::edge_index, graph_);
```

Comment:
Yeah thi new copy of edge_index_map_t does not make sense as it is not used but wasn't this supposed to get reference/pointer rather than copy and set the values for weight_map, which is part of graph_ ?

Reply:
I do not know, I can just comment this code, so it will stays there?